### PR TITLE
HomeWork5

### PR DESCRIPTION
**Pull request by VictorTuricenco**

### CODE REVIEW

File: src/main/java/org/example/homework_nr_5/Weekday.java [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+    MONDAY,
+    TUESDAY,
+    WEDNESDAY,
+    THURSDAY,
+    FRIDAY,
+    SATURDAY,
+    SUNDAY;
+
+//    public boolean isWeekday() {
```
Comment (gh3rtzz) [inline src/main/java/org/example/homework_nr_5/Weekday.java]: remove this comment if you do not need it

**Review by gh3rtzz — CHANGES_REQUESTED** [review]
remove unnecessary comments

**Comment by VictorTuricenco** [comment]
> remove unnecessary comments

Done as requested

**Review by gh3rtzz — APPROVED** [review]
approved

**Pull request merged**